scripts/cleanup-bcd-overrides.ts

```diff
@@ -0,0 +1,105 @@
+import { setLogger } from "compute-baseline";
+import { Compat, Feature } from "compute-baseline/browser-compat-data";
+import { fdir } from "fdir";
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { isDeepStrictEqual } from "node:util";
+import winston from "winston";
+import YAML from "yaml";
+import yargs from "yargs";
+import {checkForStaleCompat} from "./dist.ts";
```

Comment:
```suggestion
import { checkForStaleCompat } from "./dist";
```

It looks like the typecheck tests aren't getting run on pull requests or on `npm run test`. You can run `npm run test:types` for now and I'll open a PR to include that in `npm run test`.